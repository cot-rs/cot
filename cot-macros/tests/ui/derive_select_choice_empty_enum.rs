use cot_macros::SelectChoice;

#[derive(SelectChoice)]
enum EmptyEnum {}

fn main() {}
