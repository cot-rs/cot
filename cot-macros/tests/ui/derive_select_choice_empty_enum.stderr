error: `SelectChoice` cannot be derived for empty enums
 --> tests/ui/derive_select_choice_empty_enum.rs:3:10
  |
3 | #[derive(SelectChoice)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `SelectChoice` (in Nightly builds, run with -Z macro-backtrace for more info)
